dev_guide/deployments.adoc

```diff
@@ -247,7 +247,7 @@ triggers:
 === Image Change Trigger
 
 The ImageChange trigger results in a new deployment whenever the value of an
-image stream tag changes.
+image stream tag changes either by a build or because it was imported.
```

Comment:
which still leaves me thinking "automatic" is the wrong name for that field (should be something like "initialOnly" or something) but i guess that shipped has sailed.  

/me waves to ship in distance.